sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1624,33 +1595,96 @@ def get_transceiver_status(self):
+            rx_output_status_dict = self.get_rx_output_status()
+            trans_status['rxoutput_status_hostlane1'] = rx_output_status_dict['RxOutputStatus1']
+            trans_status['rxoutput_status_hostlane2'] = rx_output_status_dict['RxOutputStatus2']
+            trans_status['rxoutput_status_hostlane3'] = rx_output_status_dict['RxOutputStatus3']
+            trans_status['rxoutput_status_hostlane4'] = rx_output_status_dict['RxOutputStatus4']
+            trans_status['rxoutput_status_hostlane5'] = rx_output_status_dict['RxOutputStatus5']
+            trans_status['rxoutput_status_hostlane6'] = rx_output_status_dict['RxOutputStatus6']
+            trans_status['rxoutput_status_hostlane7'] = rx_output_status_dict['RxOutputStatus7']
+            trans_status['rxoutput_status_hostlane8'] = rx_output_status_dict['RxOutputStatus8']
+            tx_fault = self.get_tx_fault()
+            trans_status['txfault'] = tx_fault[0]
+            tx_los = self.get_tx_los()
+            trans_status['txlos_hostlane1'] = tx_los[0]
+            trans_status['txlos_hostlane2'] = tx_los[1]
+            trans_status['txlos_hostlane3'] = tx_los[2]
+            trans_status['txlos_hostlane4'] = tx_los[3]
+            trans_status['txlos_hostlane5'] = tx_los[4]
+            trans_status['txlos_hostlane6'] = tx_los[5]
+            trans_status['txlos_hostlane7'] = tx_los[6]
+            trans_status['txlos_hostlane8'] = tx_los[7]
+            tx_lol = self.get_tx_cdr_lol()
+            trans_status['txcdrlol_hostlane1'] = tx_lol[0]
+            trans_status['txcdrlol_hostlane2'] = tx_lol[1]
+            trans_status['txcdrlol_hostlane3'] = tx_lol[2]
+            trans_status['txcdrlol_hostlane4'] = tx_lol[3]
+            trans_status['txcdrlol_hostlane5'] = tx_lol[4]
+            trans_status['txcdrlol_hostlane6'] = tx_lol[5]
+            trans_status['txcdrlol_hostlane7'] = tx_lol[6]
+            trans_status['txcdrlol_hostlane8'] = tx_lol[7]
+            rx_los = self.get_rx_los()
+            trans_status['rxlos'] = rx_los[0]
+            rx_lol = self.get_rx_cdr_lol()
+            trans_status['rxcdrlol'] = rx_lol[0]
+            config_status_dict = self.get_config_datapath_hostlane_status()
+            trans_status['config_state_hostlane1'] = config_status_dict['ConfigStatusLane1']
+            trans_status['config_state_hostlane2'] = config_status_dict['ConfigStatusLane2']
+            trans_status['config_state_hostlane3'] = config_status_dict['ConfigStatusLane3']
+            trans_status['config_state_hostlane4'] = config_status_dict['ConfigStatusLane4']
+            trans_status['config_state_hostlane5'] = config_status_dict['ConfigStatusLane5']
+            trans_status['config_state_hostlane6'] = config_status_dict['ConfigStatusLane6']
+            trans_status['config_state_hostlane7'] = config_status_dict['ConfigStatusLane7']
+            trans_status['config_state_hostlane8'] = config_status_dict['ConfigStatusLane8']
```

Comment:
Please condense these lines to few lines

Reply:
done